include/items.php

```diff
@@ -167,6 +168,14 @@ function add_page_info_data($data) {
 
 	$text = "[attachment type='".$data["type"]."'";
 
+	if ($data["text"] == "") {
+		$data["text"] = $data["title"];
+	}
+
+	if ($data["text"] == "") {
+		$data["text"] = $data["url"];
+	}
+
 	if ($data["url"] != "")
```

Comment:
That would be alright for me. For me it is always boring to do such things, but if I had corrected such stupid brackets in new block,  it is easier, just to have the same look in the rest of the block. And I would not have mentioned it in the 1st place, if the diff was not long enough to the the next bit of the block below your changes.